java/examples/src/main/java/com/nedap/retail/services/examples/ArticleExample.java

```diff
@@ -11,25 +12,51 @@
 import com.nedap.retail.messages.article.Size;
 
 public class ArticleExample {
-    public static void runExample(final Client client) {
-        System.out.println("*** Article API example ***");
 
-        // this is our example data
-        final List<Article> exampleArticles = new ArrayList<>();
-        exampleArticles.add(createExampleArticle());
-        // we could add more articles here
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final String TAB = "\t";
+    private static final String DOUBLE_TAB = "\t\t";
+    private static final String COLON = ": ";
+    private static final String WHITESPACE = " ";
+    private static final String DOT = ". ";
+    private static final String COMMA = ", ";
```

Comment:
Lot of constants are reused, maybe you can create some class which will hold only constans and later common functionality regarding formatting of text and reuse code in examples (`OutputFormatter` or `PrintHelper` or something like that)